Sources/ArcGISToolkit/Components/FormView/FormViewModel.swift

```diff
@@ -16,53 +16,41 @@ import ArcGIS
 import Combine
 import SwiftUI
 
-/// - Since: 200.3
+/// - Since: 200.4
 public class FormViewModel: ObservableObject {
-    /// The geodatabase which holds the table and feature being edited in the form.
-    @Published private var database: ServiceGeodatabase?
-    
-    /// The featured being edited in the form.
-    @Published private(set) var feature: ArcGISFeature?
-    
-    /// The service feature table which holds the feature being edited in the form.
-    @Published private var table: ServiceFeatureTable?
-    
     /// The feature form.
-    @Published var featureForm: FeatureForm?
+    private(set) var featureForm: FeatureForm
     
     /// The current focused element, if one exists.
-    @Published var focusedElement: FormElement?
+    @MainActor @Published var focusedElement: FormElement?
     
     /// The expression evaluation task.
-    var evaluateTask: Task<Void, Never>? = nil
+    private var evaluateTask: Task<Void, Never>?
     
     /// The group of visibility tasks.
     private var isVisibleTasks = [Task<Void, Never>]()
     
     /// The list of visible form elements.
-    @Published var visibleElements = [FormElement]()
+    @MainActor @Published var visibleElements = [FormElement]()
     
-    /// Initializes a form view model.
-    public init() {}
+    /// The list of expression evaluation errors.
+    @MainActor @Published var expressionEvaluationErrors = [FormExpressionEvaluationError]()
+    
+    /// A Boolean value indicating whether evaluation is running.
+    @MainActor @Published var isEvaluating = true
     
-    /// Prepares the feature for editing in the form.
-    /// - Parameter feature: The feature to be edited in the form.
-    public func startEditing(_ feature: ArcGISFeature, featureForm: FeatureForm) {
-        self.feature = feature
+    /// Initializes a form view model.
+    /// - Parameter featureForm: The feature form defining the editing experience.
+    public init(featureForm: FeatureForm) {
         self.featureForm = featureForm
-        if let table = feature.table as? ServiceFeatureTable {
-            self.database = table.serviceGeodatabase
-            self.table = table
-        }
-        visibleElements = []
     }
     
     deinit {
         clearIsVisibleTasks()
+        evaluateTask?.cancel()
     }
     
     func initializeIsVisibleTasks() {
```

Comment:
Perhaps make this the doc for the method and remove it from the body?

```suggestion
    /// Kick off tasks to monitor `isVisible` for each element.
    func initializeIsVisibleTasks() {
```